Tests/SwiftSyntaxTest/IdentifierTests.swift

```diff
@@ -0,0 +1,24 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftSyntax
+import XCTest
+
+class IdentifierTests: XCTestCase {
```

Comment:
I got a bit lost on where to put tests exactly so created two new files as the existing places didn't seem right, more than happy to move the tests to somewhere else if there's a better place though

Reply:
Does this include moving [Tests/SwiftSyntaxTest/TokenSyntaxTests.swift](https://github.com/apple/swift-syntax/pull/2576/files#diff-d30dc020c6661e25b14357d63b64979f3452868c9c3917fc60ed9d40ee429c50) to here too? Or keep them separate?

Reply:
Yes, I would merge the two files you created.